.travis.yml

```diff
@@ -9,5 +9,13 @@ env:
   - PUPPET_VERSION=4.6 ONLY_OS=redhat-7-x86_64,centos-7-x86_64
 matrix:
   fast_finish: true
+  include:
```

Comment:
This also needs a change in ```.sync.yml``` to prevent modulesync from overwriting it. See https://github.com/Katello/puppet-pulp/blob/1ad10c1e567f165aeaef847ffde23619a2648e34/.sync.yml#L2-L4